vnext/ReactUWP/LocalByteCodeManager.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include "LocalByteCodeManager.h"
+#include "unicode.h"
+#include <winrt/Windows.Storage.h>
+
+#if _MSC_VER <= 1913
+// VC 19 (2015-2017.6) cannot optimize co_await/cppwinrt usage
+#pragma optimize( "", off )
+#endif
+
+using namespace winrt;
+
+namespace react { namespace uwp {
+
+std::future<winrt::Windows::Foundation::DateTime> LocalByteCodeManager::LoadCreatedDateTimeAsync()
+{
+  auto folder = Windows::Storage::ApplicationData::Current().LocalCacheFolder();
+  co_await winrt::resume_background();
+
+  try
+  {
+    auto file = co_await folder.GetFileAsync(L"app.bytecode");
+    return file.DateCreated();
+  }
+  catch (winrt::hresult_error const& ex)
+  {
+    winrt::Windows::Foundation::DateTime date;
```

Comment:
what date is this? is it now or is it 0?

Reply:
just want to clarify to make sure that this doesn't return dateTime::now and indicate that the bytecode is fresh